core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -441,7 +446,7 @@ public static int exists(FilesFacade ff, Path path, CharSequence root, CharSeque
         path.of(root).concat(name, lo, hi).$();
         if (ff.exists(path)) {
             // prepare to replace trailing \0
-            if (ff.exists(path.chop$().concat(TXN_FILE_NAME).$())) {
+            if (ff.exists(path.concat(TXN_FILE_NAME).$())) {
```

Comment:
`chop$()` seems to have been needed here because of:

![image](https://user-images.githubusercontent.com/7276403/203366160-0d5154c4-c444-4adf-b5ff-4816d0853a94.png)